build/deploy/mkimage.sh

```diff
@@ -13,19 +13,16 @@ set -ex
 cd -P "$(dirname $0)"
 DIR=$(pwd -P)
 
-rm -rf resources ui cockroach
-mkdir -p build
+rm -rf build && mkdir -p build
 docker run -v "$(pwd)/build":/build "cockroachdb/cockroach-dev" shell "cd /cockroach && \
   rm -rf /build/*
   make clean build testbuild >/dev/null 2>&1 && \
   find . -name '*.test' -type f -printf "\"/build/%h \"" | xargs mkdir -p && \
   find . -name '*.test' -type f -exec mv {} "/build/{}" \; && \
-  cp -r resources /build/resources && \
-  cp -r ui /build/ui && \
+  cp -r {ui,resources} /build/ && \
   cp cockroach /build/cockroach"
 
-cp -r build/resources resources
-cp -r build/ui ./ui
-cp build/cockroach cockroach
 docker build -t cockroachdb/cockroach .
 docker run -v "${DIR}/build":/build cockroachdb/cockroach
+# Files in ./build may belong to root, so let's delete that folder via this hack.
+docker run -v "$(pwd)/build":/build "cockroachdb/cockroach-dev" shell "rm -rf /build/*"
```

Comment:
I assume this is because you're developing on linux and `docker` is run as root, correct?